pkg/kubelet/network/exec/exec.go

```diff
@@ -29,6 +29,9 @@ limitations under the License.
 //         - setup, called after the infra container of a pod is
 //                created, but before other containers of the pod are created
 //         - teardown, called before the pod infra container is killed
+//         - status, called at regular intervals and is supposed to return a json
+//                formatted output indicating the pod's IPAddresses
+//                e.g. { "IPAddress" : "10.20.30.40" }
```

Comment:
If we're passing around JSON blobs, we probably need to version them, at least.  I don't know if we have any other examples of this in the system yet, so I want to be careful to set good precedent.